src/tools/pbs_probe.c

```diff
@@ -2244,7 +2244,7 @@ pbsdotconf(char * path)
 static	int
 get_realpath_values(struct infrastruct *pinf)
 {
-	char real[MAXPATHLEN];
+	char real[PATH_MAX+1];
```

Comment:
According to the realpath(3) man page...
> The resulting pathname is stored as  a  null-terminated  string, up to a maximum of PATH_MAX bytes, in the buffer pointed to by resolved_path.
So you don't need the +1.
If the Windows version of realpath() from src/lib/libWin/util.c were beefed up, we wouldn't need to preallocate the variable "real" and could just use NULL instead. Later we would have to free the allocated space. I think it would be more platform agnostic, but I'll leave the decision up to you.

Reply:
Yes you are right +1 is not necessary. Thanks. I am going with the idea of enhancing the windows function. Although right now the windows version is not used anywhere, once the enhancement's done a few places can be refactored and we can get ride of some ugly `#ifdef`s.